packages/codemod/package.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Can we make this `"type": "module"`? I'm trying to ensure that going forward new packages are ESM (eventually they will all be converted).

Reply:
Yeah, I wanted that too, but jscodeshift is not compatible with ESM. Some discussion here: https://github.com/facebook/jscodeshift/issues/500.

I was thinking of ditching jscodeshift - it seems like we only use it for imports.